middlewares/errorHandlers.js

```diff
@@ -0,0 +1,18 @@
+const { join } = require('path');
+const logger = require(join(__dirname, '..', 'logger.js'));
```

Comment:
why not to get context here as well

Reply:
Why do we need context here if the logging logic is in the logger file?